File: examples/matlab/statement_tokens.m
function result = classify(value, offset)
  result = value + offset;
  if result == 42
    result = 42;
  elseif result < 0
    result = 0;
  else
    result = 1;
  end
end

text = 'a=b:c';
answer = classify(40, 2);
for index = 1:2
  if index == 2
    disp(answer)
  end
end

File: examples/python/statement_tokens.py
def classify(value, offset,) -> int:
    result = value + offset
    if result == 42:
        return result
    elif result < 0:
        return 0
    else:
        return 1


text = "a=b:c"
answer = classify(40, 2)
if text == "a=b:c":
    print(answer)
